src/app/models/Item.php

```diff
@@ -341,7 +341,10 @@ public function matches($text)
 
         return $this->symbol == $text ||
         stristr($this->id, $text) ||
-        stristr($this->name, $text);
+        stristr($this->name, $text) ||
+        array_filter($this->data->qualities, function($q) {
```

Comment:
I also have no idea what I'm doing sometimes, but it looks like ```use ($text)``` is necessary to make use of the variable inside the function. Also, q[0] appears to be enough to allow a string comparison with the search text.
```suggestion
        array_filter($this->data->qualities,
        function ($q) use ($text) {
            return stristr($q[0], $text);
        }
    );
```